core/mgmt/v1beta/metric.proto

```diff
@@ -180,18 +176,22 @@ message ListCreditConsumptionChartRecordsRequest {
   // sequence of decimal numbers, each with optional fraction and a unit
   // suffix, such as "300ms", "1.5h" or "2h45m".
   // The minimum (and default) window is 1h.
-  optional string aggregation_window = 2 [(google.api.field_behavior) = OPTIONAL];
+  optional string aggregation_window = 2;
   // Beginning of the time range from which the records will be fetched.
-  optional google.protobuf.Timestamp start = 3 [(google.api.field_behavior) = OPTIONAL];
+  // The default value is the current timestamp.
+  optional google.protobuf.Timestamp start = 3;
   // End of the time range from which the records will be fetched.
-  optional google.protobuf.Timestamp stop = 4 [(google.api.field_behavior) = OPTIONAL];
+  // The default value is the beginning of the current day, in UTC.
+  optional google.protobuf.Timestamp stop = 4;
```

Comment:
ℹ️ For now we don't need filtering so adding it now might be confusing. In the future we might want to filter credit consumption by e.g. pipeline ID. A filter can allow the client select a list of pipeline IDs and the results will be grouped by pipeline.